doc/ref_arch/openstack/chapters/chapter04.rst

```diff
@@ -1773,8 +1645,37 @@ may need to be made for certain Edge clouds depending upon their
 resource capabilities. It should be noted that none of these changes
 affect the definition of OpenStack flavors.
 
-Edge Cloud Deployment
-^^^^^^^^^^^^^^^^^^^^^
+The previous section listed the OpenStack services deployed on the controller
+nodes depending upon the control plane distribution. As specified earlier in
+this chapter, at least 3 controller nodes should be deployed  for HA.  Compute
```

Comment:
There are some extra white spaces in this line. Probably good to fix if there's anything else that needs a new commit in this PR.

Reply:
@ildikov Thanks - Fixed.